src/ralph/accounts/ldap.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# max-complexity=25
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+
+from dj.choices import Country
+
+
+logger = logging.getLogger(__name__)
+try:
+    from django_auth_ldap.config import ActiveDirectoryGroupType
+    from django_auth_ldap.backend import (
+        populate_user,
+        LDAPSettings,
+    )
+except ImportError:
+    logger.debug("django_auth_ldap package not provided")
+else:
+    from django.dispatch import receiver
+    from django.conf import settings
+    from django.utils.encoding import force_unicode
+
+    # Add default value to LDAPSetting dict. It will be replaced by
+    # django_auth_ldap with value in settings.py and visible for each
+    # ldap_user.
+    LDAPSettings.defaults['GROUP_MAPPING'] = {}
+    GROUP_CACHE_TIMEOUT = 1200
+
+    @receiver(populate_user)
+    def staff_superuser_populate(sender, user, ldap_user, **kwargs):
+        user.is_superuser = 'superuser' in ldap_user.group_names
+        user.is_staff = 'staff' in ldap_user.group_names
```

Comment:
we should consider one group for active users (in NG, to successfully log-in you have to be in staff group)